tests/gaslessTransaction.test.ts

```diff
@@ -0,0 +1,22 @@
+import {GasLessTransaction,VerifiedWallet, Provider} from '../src'
+// Only for testing
+const apiKey ='a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799'
+const rpcUrl = "https://api.stackup.sh/v1/node/a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799"
+const paymasterUrl = 'https://api.stackup.sh/v1/paymaster/a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799'
+
+describe("userop gasless transaction", () => {
+
+    test('gasless transaction should succeed', async() => { 
+        const mnemonics = await VerifiedWallet.generateMnemonic()
+        const wallet = VerifiedWallet.importWallet(mnemonics)
+        const signer = wallet.setProvider(
+          Provider.stackUpProvider(rpcUrl)
+      )
+        const transaction = new GasLessTransaction({ apiKey,paymasterUrl,wallet, signer});
```

Comment:
@herberthk 

The test should initialize a sdk contract that should use VerifiedContract.callContract() to use userops to make a gas less call. No need to call GasLessTransaction which is not required at all.